frontend/src/components/edit-profile/edit-profile.tsx

```diff
@@ -102,6 +119,25 @@ export const EditProfile: React.FC<EditProfileProps> = ({ onClose }) => {
     await deleteUserProfile();
   };
 
+  const handleChangeAvatar = (e: React.ChangeEvent<HTMLInputElement>): void => {
+    const fileList = e.target.files;
+    if (!fileList) return;
+
+    const file = fileList[0];
+    const validation = ['image/jpeg', 'image/jpg', 'image/png'];
+    const fileSize = Math.round(file.size / 1024);
+    if (!validation.includes(file.type)) {
+      alert('This is not an Image File!');
+    } else if (fileSize > 10240) {
+      alert('File too Big, please select a file less than 10mb');
+    }
```

Comment:
Maybe it's better to move the validation rules to shared folder?
And use them both on the backend and on the frontend.